examples/crud_example.go

```diff
@@ -9,9 +9,9 @@ List users:
 	curl -X GET http://localhost:31415/v0/users
 
 List paginated users:
-	curl -X GET http://localhost:31415/v0/users?page[offset]=0&page[limit]=2
+	curl -X GET 'http://localhost:31415/v0/users?page\[offset\]=0&page\[limit\]=2'
```

Comment:
`bash` and/or `curl` on Linux seem to not like unescaped square brackets in URLs.